settings.gradle

```diff
@@ -1,6 +1,7 @@
 pluginManagement {
     plugins {
         id 'org.octopusden.octopus-release-management' version '2.0.27'
```

Comment:
What for do we octopus-release-management plugin here?

Reply:
Honestly I'm not sure what this plugin used for. It is already existed before. Probably, I can try to remove it and see how it behaves.

Reply:
I think it cannot be removed. Although the plugin is not explicitly used within the project, it seems to be required for release-related tasks. For instance, when I remove the plugin and attempt to build and publish the project to my local repository, the version defaults to "unspecified" instead of "1.0-SNAPSHOT". Therefore, I'm unsure if this plugin can be safely removed.